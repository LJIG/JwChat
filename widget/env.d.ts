/*
 * @Author            : Bian <389701057@qq.com>
 * @Date              : 2025-06-24 22:44:04
 * @LastEditors       : LJIG <389701057@qq.com>
 * @LastEditTime      : 2025-06-24 22:44:05
 * @FilePath          : /packages/env.d.ts
 * @Description       : 
 * Copyright (c) 2025 by Bian <389701057@qq.com>, All Rights Reserved. 
 */

declare module "*.vue" {
  import { DefineComponent } from "vue";
  const component: DefineComponent<{}, {}, any>;
  export default component;
}

declare module "*.md" {
  import { ComponentOptions } from "vue";
  const Component: ComponentOptions;
  export default Component;
}

declare module "wechat-emoji-parser";

declare module "@/utils/scroll";

declare module "*.scss" {
  const classes: { readonly [key: string]: string };
  export default classes;
}

declare module "*.png";
declare module "@/utils/imgUrl";
