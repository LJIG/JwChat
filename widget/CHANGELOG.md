## 0.0.2 (2022-04-22)


### Bug Fixes

* fix template ([21a31e4](https://github.com/jrainlau/MY-Kit/commit/21a31e41647fadef0e458539476805c28bd59e69))


### Features

* chat组件 ([e4abb9a](https://github.com/jrainlau/MY-Kit/commit/e4abb9a5a12b785b4ff23e843e37263e32ffca17))


### BREAKING CHANGES

* 突破从无到有



