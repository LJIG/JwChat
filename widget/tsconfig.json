{
  "compilerOptions": {
    "noImplicitAny": false, // 是否在表达式和声明上有隐含的any类型时报错
    "target": "esnext",
    "useDefineForClassFields": true,
    "module": "esnext",
    "moduleResolution": "node",
    "strict": true,
    "jsx": "preserve",
    "sourceMap": true,
    "resolveJsonModule": true,
    "esModuleInterop": true,
    "allowJs": true,
    "lib": ["esnext", "dom"],
    "skipLibCheck": true,
    "paths": {
      "packages/*": ["./packages/*"]
    }
  },
  "include": [
    "*.ts",
    "packages/**/*.ts",
    "packages/**/*.d.ts",
    "packages/**/*.tsx",
    "packages/**/*.vue"
  ]
}
