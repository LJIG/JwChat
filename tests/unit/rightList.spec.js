import '../util.js'

import { mount  } from '@vue/test-utils'
import widget from '@/components/RightList/index.vue'

describe('RightList 组件', () => {

  const config = {
    tip:'',
    notice:'',
    listTip:'',
    list: [
      // {
      //   name: 'JwChat',
      //   "img": "image/three.jpeg"
      // },
      // {
      //   name: 'JwChat',
      //   "img": "image/three.jpeg"
      // },
      // {
      //   name: '留恋人间不羡仙',
      //   "img": "image/one.jpeg"
      // },
      // {
      //   name: '只盼流星不盼雨',
      //   "img": "image/two.jpeg"
      // }
    ]
  }


  const wrapper = mount(widget, { propsData: { config } })
  // 使用 jest 我们可以创建HTML输出的截图文件
  expect(wrapper.html()).toMatchSnapshot()
})
