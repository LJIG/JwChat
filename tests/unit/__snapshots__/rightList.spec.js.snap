// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div class="wrapper">
  <div class="notice">
    <div class="empty empty" style="width: 8rem;"><i class="iconfont icon-empty" style="font-size: calc(8rem / 2);"></i></div>
    <p style="padding-left: 0.2rem;">群公告:</p>
    <p style="font-size: 0.8rem; margin-top: 0.5rem; padding: 0px .2rem;"></p>
  </div>
  <div class="userList">
    <div>组成员 (0)</div>
    <ul></ul>
  </div>
</div>
`;
